### PR TITLE
Runtime 43

### LINKED ISSUE #41: Update rhythmbox dependencies to more recent versions
Package | Current version | Recent version
------------ | ------------- | -------------
tdb |    1.4.2   | 1.4.3
libpeas | 1.24.0 | 1.28.0
totem-pl-parser | 3.26.2 | 3.26.5
libdmapsharing | 2.9.39 |  2.9.41
libgee | 0.20.0 | 0.20.3
avahi | 0.7 | 0.8

### LINKED ISSUE #60: Consider upgrading runtime org.gnome.Platform version
```
Info: runtime org.gnome.Platform branch 42 is end-of-life, with reason:
   The GNOME 42 runtime is no longer supported as of March 21, 2023. Please ask your application developer to migrate to a supported platform.
```
Changing following line should be just enough.
https://github.com/flathub/org.gnome.Rhythmbox3/blob/8fc7d9b86fcd57690b5da16c55d690698c4f6224/org.gnome.Rhythmbox3.json#L4

### PR DESCRIPTION
**Pull request by bbhtt**
Closes https://github.com/flathub/org.gnome.Rhythmbox3/issues/60
Closes https://github.com/flathub/org.gnome.Rhythmbox3/issues/41

### CODE REVIEW

File: org.gnome.Rhythmbox3.json [hunk 1]
```diff
@@ -43,13 +43,13 @@
         {
             "name": "grilo",
             "buildsystem": "meson",
-            "config-opts": [ "-Denable-gtk-doc=false", "-Denable-grl-pls=false" ],
+            "config-opts": [ "-Denable-gtk-doc=false", "-Denable-grl-pls=false", "-Dsoup3=false" ],
```
Comment (hfiguiere) [inline org.gnome.Rhythmbox3.json:46]: I think you mean true here.

**Comment by hfiguiere** [comment]
GNOME 44 is out

Grilo needs to be built with libsoup3.

(and possibly more)

File: org.gnome.Rhythmbox3.json (hunk 1 above)
Comment (bbhtt) [reply org.gnome.Rhythmbox3.json:46]: But rhythmbox still depends on libsoup-2.4

**Comment by hfiguiere** [comment]
bot, build

File: org.gnome.Rhythmbox3.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "org.gnome.Rhythmbox3",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "42",
+    "runtime-version": "43",
```
Comment (hfiguiere) [inline org.gnome.Rhythmbox3.json:4]: 44 is out.

Comment (bbhtt) [reply org.gnome.Rhythmbox3.json:4]: 44 has libsoup3 only no libsoup2

**Comment by hfiguiere** [comment]
submitted https://github.com/flathub/shared-modules/pull/242 for libsoup-2.4

**Comment by A6GibKm** [comment]
Is the app using soup2 directly? If not the path forward should be to use soup3 on all modules.

**Comment by hfiguiere** [comment]
yes it does require it. Or get rid of a lot of user functionalities.

**Comment by hfiguiere** [comment]
See #62

**Pull request closed without merging**